File: in-motion-server-java/src/main/java/com/inmotion/inmotionserverjava/model/UserInfoDto.java
package com.inmotion.inmotionserverjava.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserInfoDto {
    private String id;
    private String email;
    private String nickname;
    private String token;
}

File: in-motion-android/app/src/main/java/com/inmotion/in_motion_android/data/dto/auth/UserInfoDto.kt
package com.inmotion.in_motion_android.data.dto.auth

import com.inmotion.in_motion_android.database.entity.UserInfo

data class UserInfoDto(
    val id: String,
    val email: String,
    val nickname: String,
    val token: String
) {
    fun toUserInfo(): UserInfo {
        return UserInfo(id, email, nickname, token)
    }
}
